assets/prometheus-k8s/rules.yaml

```
@@ -380,6 +380,13 @@ spec:
       record: node_role_os_version_machine:cpu_capacity_sockets:sum
     - expr: clamp_max(sum(alertmanager_notifications_total),1)
       record: cluster:alertmanager_routing_enabled:max
+    - alert: ClusterOOMWarning
+      annotations:
+        message: Cluster is experiencing out of memory (OOM) kills
```

Can we have something like:
```suggestion
        message: Pod {{ $labels.namespace }}/{{ $labels.pod }} ({{ $labels.container }}) is experiencing out of memory termination. 
```

---

Yes. This makes a ton of sense!